samples/jigsaw_puzzle/jigsaw-prototype.py

```diff
@@ -46,8 +47,14 @@
             ExtraArgs={"ContentType": "image/png", "ACL": "public-read"},
         )
         s3_key_counter += 1
+        filenames.append(
+            "https://dcc-bp-public-dev.s3.us-east-2.amazonaws.com/"
+            + s3_key_prefix
+            + str(s3_key_counter)
+            + ".png"
+        )
 
-
+print(filenames)
```

Comment:
Remove?

Reply:
It is needed for jigsaw puzzles, the script isn't part of the main program, but a script we (devs) can use to quickly get a jigsaw onto s3 and get the filenames :)